recipes/cibots/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+    name: cibots
+    version: 0.1
+
+source:
+    url: https://github.com/UV-CDAT/ci-bots/archive/0.1.tar.gz
+    sha256: c16d96619efa86fdabc23f49b6bff04a78e1d99a097a5fa8320504038dd6976b
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
not really except non of my users or i have access to windows. I can turn it on if you want.